vllm/v1/engine/core_client.py

```diff
@@ -153,12 +154,23 @@ def get_output(self) -> List[EngineCoreOutput]:
 
         return engine_core_outputs
 
-    def add_request(self, request: EngineCoreRequest) -> None:
-
-        self.input_socket.send_multipart((self.encoder.encode(request), ),
+    def send_input(self,
```

Comment:
since this is a private method
```suggestion
    def _send_input(self,
```